starknet_py/contract.py

```diff
@@ -400,7 +400,7 @@ async def from_address(
 
         :raises ContractNotFoundError: when contract is not found
         :param address: Contract's address
-        :param client: Client, WARNING: This method does not work with FullNodeClient!
+        :param client: Client
```

Comment:
Perhaps we could do something like `if isinstance(client, FullNodeClient): raise`. I know it won't catch the cases where AccountClient with FullNodeClient inside is used but it's better than nothing.